docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc

```diff
@@ -47,37 +47,34 @@ property in Undertow subsystem:
 [[proxies]]
 === Proxies
 
-By default, mod_cluster is configured for multicast-based discovery. To
-specify a static list of proxies, create a remote-socket-binding for
-each proxy and then reference them in the 'proxies' attribute. See the
-following example for configuration in the domain mode:
+By default, mod_cluster is configured for discovery based on UDP Multicast.
+To specify a static list of proxies, create a `remote-socket-binding` for each proxy and then reference them by the `proxies` attribute.
+See the following configuration example:
 
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] /socket-binding-group=ha-sockets/remote-destination-outbound-socket-binding=proxy1:add(host=10.21.152.86, port=6666)
-{
-    "outcome" => "success",
-    "result" => undefined,
-    "server-groups" => undefined
-}
-[domain@localhost:9990 /] /socket-binding-group=ha-sockets/remote-destination-outbound-socket-binding=proxy2:add(host=10.21.152.87, port=6666)
+[standalone@localhost:9990 /] /socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy1:add(host=10.21.152.86, port=6666)
 {
     "outcome" => "success",
-    "result" => undefined,
-    "server-groups" => undefined
+    "response-headers" => {"process-state" => "reload-required"}
```

Comment:
This op doesn't put the server in reload-required. The preceding section's setting of the instance-id does, but it's not clear this section's examples are meant to continue on from the preceding section.

In any case, it's probably better for the instance-id example to include a 'reload' step.

Reply:
Ah of course. Added reload to the example there.